ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -1593,3 +1594,36 @@ def expand(
         Output Array
     """
     return ivy.current_backend(x).expand(x, shape, out=out)
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_function
+def ConcatFromSequence(
```

Comment:
Please update the name of the function to `concat_from_sequence`. We do not follow ONNX's naming convention (I just left a comment on the ToDo list as well to avoid future confusion).